app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -101,6 +104,10 @@ class ConversationViewModel @Inject constructor(
                 conversationViewState = conversationViewState.copy(conversationName = conversationName)
             }
         }
+
+        viewModelScope.launch {
+            markMessagesAsNotified(conversationId!!, System.currentTimeMillis().toStringDate()) //TODO Failure is ignored
+        }
```

Comment:
Any reason why we are not including this thread launch into the `init` block?

Reply:
Maybe it would be better to move these coroutines launches to separated functions to improve readability? :)